src/current/_includes/molt/migration-schema-design-practices.md

```diff
@@ -0,0 +1,7 @@
+Follow these recommendations when converting your schema for compatibility with CockroachDB.
+
+- Define an explicit primary key on every table. For more information, refer to [Primary key best practices]({% link {{ site.current_cloud_version }}/schema-design-table.md %}#primary-key-best-practices).
+
+- Do **not** use a sequence to define a primary key column. Instead, Cockroach Labs recommends that you use [multi-column primary keys]({% link {{ site.current_cloud_version }}/performance-best-practices-overview.md %}#use-multi-column-primary-keys) or [auto-generating unique IDs]({% link {{ site.current_cloud_version }}/performance-best-practices-overview.md %}#use-functions-to-generate-unique-ids) for primary key columns.
```

Comment:
@Jeremyyang920 should this explain why exactly? Mainly because of the hot spotting problem. Might be worthwhile to provide reasoning for this statement instead of a blanket statement

Reply:
Also, I spotted some internal convos suggesting that we shouldn't apply this blanket rule anymore. True or no?

Reply:
That would be true, I think we have seen some people say its okay to keep sequences. It might be best to remove this blanket rule.